app/models/facts/metric.rb

```diff
@@ -5,4 +5,6 @@ class Facts::Metric < ApplicationRecord
 
   validates :dimensions_date, presence: true
   validates :dimensions_item, presence: true
+  validates :pageviews, numericality: { only_integer: true }
```

Comment:
The validation for `only_integer: true` does not allow `nil` values. 
We should be able to store metrics facts without being explicit about the GA metrics. What about adding `allow_nil: true` ?

Reply:
Fixed with eaa10d4de322c29e1eee762c867989a8558b1a07